### PR TITLE
add scripts for wifi onboarding

### PR DESCRIPTION
**Pull request by rahul-daga-imgtec**
Signed-off-by: Rahul Daga <Rahul.Daga@imgtec.com>

### CODE REVIEW

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins

File: src/cgi-bin/save_settings.lp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+</head>
+<body id="home" class="en_GB"  onload="handleBrowser();">
+	<div class="container">
+		<noscript class="no-js-message">
+			<div>
+				<p><%= STR_JAVASCRIPT_MSG_1 %></p>
+				<p><%= STR_JAVASCRIPT_MSG_2 %></p>
+			</div>
+		</noscript>
+		<img src="/images/logo.gif" alt="Caskeid" class="logo">
```
Comment (abhijit-mahajani-imgtec) [inline src/cgi-bin/save_settings.lp]: should be Creator logo

**Comment by abhijit-mahajani-imgtec** [comment]
There is Flowcloud logo, please pull the latest changes where Creator logo and images will be picked up.

File: src/cgi-bin/index_wifi.lp [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+                    <input type="hidden" id="config_method" name="config_method" value="web_page"/>
+                </form>
+            </div>
+            <br />
+
+        </div>
+        <div id="footer">
+            <span class="span-3-5 mobile-span-1">
+                <span class="non-input">
+                    <%= "&copy; Imagination Technologies 2015" %>
```
Comment (abhijit-mahajani-imgtec) [inline src/cgi-bin/index_wifi.lp]: 2016

File: src/cgi-bin/save_settings.lp [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+			<input type="hidden" name="encryption" value="<%= cgilua.POST.encryption %>" />
+			<input type="hidden" name="addr_method" value="<%= cgilua.POST.addr_method %>" />
+			<input type="hidden" name="ip_address" value="<%= cgilua.POST.ip_address %>" />
+			<input type="hidden" name="netmask" value="<%= cgilua.POST.netmask %>" />
+			<input type="hidden" name="dns_server" value="<%= cgilua.POST.dns_server %>" />
+			<input type="hidden" name="router_ip" value="<%= cgilua.POST.router_ip %>" />
+			<input type="hidden" name="config_method" value="<%= cgilua.POST.config_method %>" />
+		</form>
+		<div id="footer">
+			<p>&copy; Imagination Technologies 2015</p>
```
Comment (abhijit-mahajani-imgtec) [inline src/cgi-bin/save_settings.lp]: new page should be 2016

**Comment by abhijit-mahajani-imgtec** [comment]
Approved and Verified.

**Pull request merged**